basic-bootc.ks.in

```diff
@@ -0,0 +1,20 @@
+# Substitute something in for REPO or this will all come crashing down.
+bootc --source-imgref=registry:quay.io/fedora-testing/fedora-bootc:rawhide-standard --stateroot=default
+
+network --bootproto=dhcp
```

Comment:
Can you use kickstart fragments from https://github.com/rhinstaller/kickstart-tests/tree/main/fragments to avoid kickstart duplication and keep it aligned with other kickstart tests?